dashboard/app/models/pd/enrollment.rb

```diff
@@ -182,7 +182,7 @@ def email=(value)
   end
 
   def resolve_user
-    user || User.find_by_email_or_hashed_email(email)
+    user || User.find_by_email_or_hashed_email(email) || (User.all.find {|u| u.email_for_enrollments == email} if deleted_at.nil?)
```

Comment:
Since an alternate email is able to become the email associated with an enrollment, it might be needed to connect the enrollment to a user. However, this raises issues when trying to delete a user as this method will still sometimes find a user when it shouldn't (since the user would have been deleted in this case), so we only want to allow this searching by `email_for_enrollments` for non-deleted users.

Reply:
Oh okay I didn't realize it wouldn't include deleted users, that works great then!

Reply:
yep it is one of those gems that you have to know about in order to be able to even find the docs. here is where this behavior comes from: https://github.com/ActsAsParanoid/acts_as_paranoid